src/components/UI/molecules/CardDivider/CardDivider.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Divider from '<atoms>/Divider/Divider';
+import Text from '<atoms>/Text/Text';
+import spacing from '<variables>/spacing';
+import { fontWeights, fontSizes } from '<variables>/fonts';
+import { textColors, borderColors } from '<variables>/colorPalette';
+
+/**
+ * @description - CardDivider Component
+ *
+ * @prop {string} text - divider text
+ * @prop {string} cardDividerMargin - divider margin
+ * @prop {string} width - maxWidth
+ * @prop {string} height - height
+ * @prop {string} dividerColor - color
+ * @prop {string} margin - margin
+ * @prop {string} fontSize - font Size
+ * @prop {string} textAlign - text Align
+ * @prop {string} textColor - color
+ * @prop {string} textDisplay - display
+ * @prop {string} textTransform - text transform
+ * @prop {string} fontWeight - fontWeight
+ * @prop {string} textPadding - text padding
+ * @prop {string} content - content
+ * @prop {string} cardDividerPadding - card divider padding
+ * @prop {string} display - container display
+ *
+ * @return {component} CardDivider
+ */
+const CardDivider = ({
+  text,
+  cardDividerMargin,
+  width,
+  height,
+  margin,
+  dividerColor,
+  fontSize,
+  textAlign,
```

Comment:
Is there a place in the design that the text is in anywhere other than the center